configmap/informed_watcher.go

```diff
@@ -51,8 +51,9 @@ func NewInformedWatcherFromFactory(sif informers.SharedInformerFactory, namespac
 func NewInformedWatcher(kc kubernetes.Interface, namespace string) *InformedWatcher {
 	return NewInformedWatcherFromFactory(informers.NewSharedInformerFactoryWithOptions(
 		kc,
-		// This is the default resync period from controller-runtime.
-		10*time.Hour,
+		// We noticed that we're getting updates all the time anyway, due to the
+		// watches being terminated and re-spawned.
```

Comment:
I'd mention that this is the resync period for the uninitiated.

Reply:
idk, it's in the API.

Reply:
I’m not seeing a re-listing & informer updates (Add/Update/Delete events) when watches time out - I just see the watch reconnect. What K8s version do you see this with?